components/rt/src/rt_estimator.m

```diff
@@ -1,7 +1,7 @@
 %% Atte Aalto
-
+% 
 % change to the root of the file
-pathToFile = fileparts(mfilename('fullpath'));
+pathToFile = "input/";
```

Comment:
`fileparts(mfilename('fullpath'))` has to stay, as it will give the location of the `src` folder